plugin/src/test/groovy/org/gradle/testretry/testframework/TestNGViaJUnitEngineFuncTest.groovy

```diff
@@ -55,9 +58,12 @@ class TestNGViaJUnitEngineFuncTest extends BaseTestNGFuncTest {
 
     @Override
     String reportedParameterizedMethodName(String gradleVersion, String methodName, String paramType, int invocationNumber, @Nullable String paramValueRepresentation) {
-        GradleVersion.version(gradleVersion) > GRADLE_5_4_1
-            ? "${methodName}(${paramType}) > [${invocationNumber}] ${paramValueRepresentation ?: ''}"
-            : "${methodName}(${paramType})[${invocationNumber}]"
+        switch (GradleVersion.version(gradleVersion)) {
+            case { it < GRADLE_6_1 }: return "${methodName}(${paramType})[${invocationNumber}]"
+            case { it < GRADLE_7_0 }: return "[${invocationNumber}] ${paramValueRepresentation ?: ''}"
+            case { it < GRADLE_8_1 && it != GRADLE_7_6_4 }: return "${methodName}(${paramType})[${invocationNumber}]"
```

Comment:
I assume that a potential Gradle 7.6.5 release would also match the pattern in line 64, but I have no idea how likely such a version would be. So it's probably not worth considering it at the moment.

Reply:
Yeah, I think we'll notice it during our CI checks and can easily fix it then.